release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -2043,9 +2043,13 @@ $ oc patch route/canary -n openshift-ingress-canary -p '{"spec":{"tls":{"termina
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1932401[*BZ#1932401*])
 
-* Updates to openvswitch ("net: openvswitch: reorder masks array based on usage") code causes the openvswitch et/openvswitch/flow_table::flow_lookup accessing per-cpu data condition on preemptible (and migratable) sections, leading to a real time kernel panic. As a result, the kernel-rt is unstable and will impact low latency applications.
+* Updates to openvswitch ("net: openvswitch: reorder masks array based on usage") code, causes the openvswitch et/openvswitch/flow_table::flow_lookup accessing per-cpu data condition on preemptible (and migratable) sections, leading to a real time kernel panic. As a result, the kernel-rt is unstable and will impact low latency applications.
 +
-It is recommended not to upgrade to {product-title} 4.7 until this is fixed.
+Customers who are using the kernel-rt and upgrade to {product-title} 4.7.0 will experience a severe impact.
```

Comment:
I thought this [was going to be][1]:

> Customers who are using the kernel-rt and run either {product-title} 4.6 or 4.7.0 will experience a severe impact.

[My point][2] is that both folks running 4.6 and 4.7 are already impacted, so this bug has nothing to do with upgrades to 4.7.  It might have something to do with updates from 4.5 to 4.6, but that's irrelevant for the 4.7-and-later release notes.

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=1918456#c37
[2]: https://bugzilla.redhat.com/show_bug.cgi?id=1918456#c29

Reply:
wking didn't have a chance to review during the initial iteration and had some important changes. Sorry about the inconvenience.